modules/openxr/openxr_api.cpp

```diff
@@ -837,6 +830,13 @@ bool OpenXRAPI::create_session() {
 		wrapper->on_session_created(session);
 	}
 
+	// Check our environment blend mode. This needs to happen after we call call `on_session_created()`
```

Comment:
`call` repeated twice.

Reply:
Ah, thanks for catching that! Updated in latest push